### PR TITLE
Ytnapari

### PR DESCRIPTION
**Pull request by chrishavlin**
new blog post showing how to use napari for visualization with yt

### CODE REVIEW

File: content/posts/ytnapariparty.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "A yt-napari party"
+date: 2021-01-12T18:22:25-06:00
+author: Chris Havlin
```
Comment (munkm) [inline content/posts/ytnapariparty.md:4]: Do you have a personal webpage? We can link it here.

File: content/posts/ytnapariparty.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+cover: /img/ytnapari/ytnapari_enzo3d.png
+categories:
+  - Tutorial
+tags:
+  - Community
```
Comment (munkm) [inline content/posts/ytnapariparty.md]:
```suggestion
  - Experimental
```

I think this should have a new tag. Community and development are more about governance and social things with the project. What do you think?

File: content/posts/ytnapariparty.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+```python
+import napari
+with napari.gui_qt(): 
+    viewer = napari.Viewer()
+    viewer.add_image(dens_log)
+```
+
+Or if in an IPython session (or Jupyter notebook), use the `%gui qt` magic command:
+
+```
```
Comment (munkm) [inline content/posts/ytnapariparty.md]: do you want this to have python formatting or is intentionally not highlighted?

**Review by munkm — APPROVED** [review]
This looks *awesome*!!! Great post!!! I have a couple of minor thoughts and am happy to merge after you reply! 

I noticed your videos are unlisted on youtube and hosted under your account. We should consider moving them to the yt-project youtube channel (or maybe we could create an "experimental" playlist that links to them?)! What do you think?

File: content/posts/ytnapariparty.md (hunk 1 above)
Comment (chrishavlin) [reply content/posts/ytnapariparty.md:4]: yes... but it's empty... if I get to updating it soon I'll edit this PR but just publish without whenever. I can always submit a PR to update it?

File: content/posts/ytnapariparty.md (hunk 2 above)
Comment (chrishavlin) [reply content/posts/ytnapariparty.md]: yes!! That's what I needed. `Experimental` is great!

File: content/posts/ytnapariparty.md (hunk 3 above)
Comment (chrishavlin) [reply content/posts/ytnapariparty.md]: yes, python formatting! will add it.

File: content/posts/ytnapariparty.md (hunk 1 above)
Comment (munkm) [reply content/posts/ytnapariparty.md:4]: Yep! Totally ok to update later! I just wanted to make sure we *did* link it if you had one.

**Review by munkm — APPROVED** [review]

**Pull request merged**